t/t2232-job-info-security.t

```diff
@@ -6,51 +6,17 @@ test_description='Test flux job info service security'
 
 test_under_flux 1 job
 
-# We have to fake a job submission by a guest into the KVS.
```

Comment:
commit message typo: "but the job-info LRU will now more aggressively caches eventlog" -- caches -> cache